CHANGELOG.md

```diff
@@ -18,11 +18,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - feat: enable remote write proxy by default [#2483]
 - feat: add dot suffix to internal dns addresses [#2502]
 - feat: make creation of default Instrumentation object for opentelemetry operator configurable [#2506]
+- chore(kube-prometheus-stack): update kube-prometheus-stac chart to 39.11.0 [#2446]
```

Comment:
Shouldn't this be in the "Breaking Changes" section?

Reply:
Should :innocent: